package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "^3.0.2",
+    "@material-ui/lab": "^3.0.0-alpha.30",
```

Comment:
Do 'npm install' on both the backend and frontend server first before you start working on stories. This is so that you have all the existing dependencies and won't run into errors.